index.html

```diff
@@ -1,75 +1,98 @@
 
-  <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
+    <link rel="stylesheet" href="css/index.css">
 
 </head>
 
 <body>
+    <div class="nav_bar">
+        <img src="./img/lambda-black.png" alt="lambda school" class="lambda-img" />
+        <nav>
+            <a href="#">Home</a>
+            <a href="./about.html">About</a>
+            <a href="#">Products</a>
+            <a href="#">Blog</a>
+            <a href="#">Contact</a>
+        </nav>
+    </div>
     <div class="container">
-   
+        <section class="intro-img"></section>
+
         <section class="top-content">
             <div class="text-container">
                 <h2>The Future</h2>
-                <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
+                <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui
+                    id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus.
+                    Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna
+                    eget finibus.</p>
             </div>
             <div class="text-container">
                 <h2>The Past</h2>
-                <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
+                <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui
+                    id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus.
+                    Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna
+                    eget finibus.</p>
             </div>
         </section>
-        
+
         <section class="middle-content">
-        
+
             <h2>Why Did It Have To Be Boxes...</h2>
-            
+
             <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
+                <div class="box box1">Box 1</div>
+                <div class="box box2">Box 2</div>
+                <div class="box box3">Box 3</div>
```

Comment:
Awesome, you understand the concept of parent elements that cascade to its children. You overrode the box colors by adding a unique class to each of them that specifies the color of the box.